certified-connectors/TxtSync/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2235 @@
+    },
+    {
+      "propertyName": "Categories",
+      "propertyValue": "Communication;Productivity"
+    }
+  ],
+  "host": "api.txtsync.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/sms/send": {
+      "post": {
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "array",
+              "items": {
+                "type": "object",
+                "properties": {
+                  "FromNumber": {
+                    "type": "string",
+                    "description": "The Sender ID or E164 number to send from",
+                    "title": "From Number",
+                    "x-ms-visibility": ""
+                  },
+                  "ToNumber": {
+                    "type": "string",
+                    "description": "The mobile number to send the SMS to",
+                    "title": "To Number",
+                    "x-ms-visibility": ""
+                  },
+                  "ContactID": {
+                    "type": "integer",
+                    "format": "int64",
+                    "description": "Unique identifier of the Contact",
+                    "title": "Contact ID",
+                    "x-ms-visibility": ""
+                  },
+                  "Direction": {
+                    "type": "integer",
+                    "format": "int32",
+                    "description": "Direction of the SMS. 0 is outbound and 1 is inbound",
+                    "title": "Direction",
+                    "x-ms-visibility": ""
+                  },
+                  "Message": {
+                    "type": "string",
+                    "description": "The message body of the message",
+                    "title": "Message",
+                    "x-ms-visibility": ""
+                  },
+                  "CreatedDate": {
+                    "type": "string",
+                    "description": "The date (UTC) of when the SMS was created",
+                    "title": "Created Date",
+                    "format": "date-time",
+                    "x-ms-visibility": ""
+                  },
+                  "SMSID": {
+                    "type": "integer",
+                    "format": "int64",
+                    "description": "Unique identifier of the SMS",
+                    "title": "SMS ID",
+                    "x-ms-visibility": ""
+                  }
+                }
+              }
+            }
+          }
+        },
+        "summary": "Send an SMS",
+        "description": "Sends out a single SMS to an E164 number",
+        "operationId": "SendSMS",
+        "parameters": [
+          {
+            "name": "x-api-key",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "default": "M3Yf9L0njL8eL9ukphahq9zk5LFRIOvn2fT4BcmV",
```

Comment:
@TxtSync We are receiving security alerts regarding secret-look-a-like values. Please remove them from the swagger.

Reply:
Applies this comment to similar properties and values in the swagger

Reply:
@TxtSync This is not resolved, please remove the property from the swagger.